src/plugins/dam/components/image-picker-field/index.jsx

```diff
@@ -1,41 +1,66 @@
-import React, { lazy, useState } from 'react';
+import React, { lazy } from 'react';
 import classNames from 'classnames';
-import { Button, Label, Media } from 'reactstrap';
-import { CreateModalButton, Icon } from 'components';
+import { Button, Label, Media, Card, CardImgOverlay, CardTitle } from 'reactstrap';
+import NodeRef from '@gdbots/pbj/well-known/NodeRef';
+import { CreateModalButton, Icon, useField, useFormContext } from 'components';
 import damUrl from 'plugins/dam/damUrl';
 import useNode from 'plugins/ncr/components/useNode';
-
-// fixme: add uploader.
+import noop from 'lodash-es/noop';
 
 const ImagePickerModal = lazy(() => import('plugins/dam/components/image-picker-field/ImagePickerModal'));
 
+const damAspectRatio = (aspectRatio) => {
+  let damAspectRatio = null;
+  if (`${aspectRatio}` === 'auto' || `${aspectRatio}` === 'original') {
```

Comment:
do we need to check for custom?

Reply:
I don't think so because this just helps return back a default value. I can rename function to `defaultAspectRatio` for clarity.